game/src/test/scala/io/aigar/model/TeamRepositorySpec.scala

```diff
@@ -0,0 +1,65 @@
+package io.aigar.model
+
+import org.scalatest._
+
+class TeamRepositorySpec extends FlatSpec {
+
+  def withInMemDatabase(testCode: (TeamRepository, List[Team]) => Any) {
+    val teamRepository = new TeamRepository
+    val team1 = teamRepository.createTeam(Team(None, "EdgQWhJ!v&", "team1", 0))
+    val team2 = teamRepository.createTeam(Team(None, "not_that_secret", "team2", 50))
+    val team3 = teamRepository.createTeam(Team(None, "xx3ddfas3", "team3", 56))
+
+    val listTeams = List(team1, team2, team3)
+
+    try{
+      testCode(teamRepository, listTeams)
+    }
+    finally {
+      teamRepository.dropSchema
```

Comment:
Why is this necessary if we are using the in-memory database? I thought it dropped the schema after the `closeConnection`, did I misunderstand?